scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -194,9 +214,16 @@ task CreateImportTsvs {
       export TMPDIR=/tmp
 
       export GATK_LOCAL_JAR=~{default="/root/gatk.jar" gatk_override}
+      ~{for_testing_only}
```

Comment:
did we talk about renaming this input to something more descriptive/specific? what is this used for?

Reply:
this is used during integration testing. we thought we might not need it since we have a way to pass in a service account, but this is doing the auth login before it's needed when using the aou service account. i'm not sure if we should remove it and then put it back in when we look at the integration tests again or leave it until we know what it should be.